frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,7 +172,8 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
-  const resultDataVolumeName = prefixedID(vmName, name);
+  // We can generate a random name every time, because this modal should not operate on disks with live datavolumes
+  const resultDataVolumeName = generateDataVolumeName(vmName, name);
```

Comment:
I found one problem here regarding consistency.

When I create a template, its DV will have `mytemplate-disk-0` name.
But when I add a DV disk later (once the template was created) here in this modal its name will be `mytemplate-disk-1-efc5z`.

Let's make this consistent with `resolveDataVolumeName` logic. Can we move this function to some utils file and use it here? We should have all the required information already here for calling it.

Reply:
Done!